developer/src/kmc-keyboard/test/fixtures/sections/layr/invalid-touch-hardware.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!DOCTYPE keyboard SYSTEM "../../../../../../../resources/standards-data/ldml-keyboards/techpreview/dtd/ldmlKeyboard.dtd">
+<keyboard locale="und" conformsTo="techpreview">
+  <names>
+    <name value="layr-invalid-form" />
+  </names>
+
+  <keys>
+    <key id="grave" to="`" />
+    <key id="one" to="1" />
+  </keys>
+
+  <layers form="hardware" hardware="us">
+    <layer id="base">
+      <!-- beware: this is mapping ` and 1! -->
+      <row keys="grave one" />
+    </layer>
+  </layers>
+  <layers form="touch" hardware="iso"> <!-- not allowed: hardware on touch -->
```

Comment:
Can we add a comment about what is invalid here for future maintainers?

Reply:
done